backend/src/main/java/io/github/projektalmanac/amoxtli/backend/service/UserService.java

```diff
@@ -1,7 +1,109 @@
 package io.github.projektalmanac.amoxtli.backend.service;
 
+import io.github.projektalmanac.amoxtli.backend.entity.User;
+import io.github.projektalmanac.amoxtli.backend.exception.*;
+import io.github.projektalmanac.amoxtli.backend.generated.model.PerfilUsuarioDto;
+import io.github.projektalmanac.amoxtli.backend.mapper.UserMapper;
+import io.github.projektalmanac.amoxtli.backend.repository.UserRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
 import org.springframework.stereotype.Service;
 
+import java.io.IOException;
+
 @Service
 public class UserService {
-}
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
+
+    private UserRepository userRepository;
+
+    @Autowired
```

Comment:
```suggestion
    private UserRepository userRepository;

    @Slf4j
    @Autowired
```

Tenemos lombock, úsalo.